From 37a8b7377545447544f4615fe0491ad8f25771ce Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Thu, 10 Jan 2013 16:22:59 +0200
Subject: [PATCH 050/184] am33xx.dtsi: Make the MUSB not crash on load

Two entries for the same musb instance end up with a crash.
Remove the duplicate, but it still doesn't work.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 arch/arm/boot/dts/am33xx.dtsi |   67 ++++-------------------------------------
 1 file changed, 6 insertions(+), 61 deletions(-)

diff --git a/arch/arm/boot/dts/am33xx.dtsi b/arch/arm/boot/dts/am33xx.dtsi
index c9eee93..9142e54 100644
--- a/arch/arm/boot/dts/am33xx.dtsi
+++ b/arch/arm/boot/dts/am33xx.dtsi
@@ -270,66 +270,6 @@
 			status = "disabled";
 		};
 
-		usb0_phy: phy0 {
-			compatible = "nop-xceiv-usb";
-		};
-
-		usb1_phy: phy1 {
-			compatible = "nop-xceiv-usb";
-		};
-
-		usb_otg_hs: usb_otg_hs {
-			compatible = "ti,musb-am33xx";
-			ti,hwmods = "usb_otg_hs";
-			multipoint = <1>;
-			num-eps = <16>;
-			ram-bits = <12>;
-			port0-mode = <3>;
-			port1-mode = <1>;
-			power = <250>;
-			usb0-phy = <&usb0_phy>;
-			usb1-phy = <&usb1_phy>;
-		};
-
-		rtc {
-			compatible = "ti,da830-rtc";
-			ti,hwmods = "rtc";
-		};
-
-		spi0: spi@48030000 {
-			compatible = "ti,omap4-mcspi";
-			ti,hwmods = "spi0";
-			#address-cells = <1>;
-			#size-cells = <0>;
-			reg = <0x48030000 0x400>;
-			interrupt-parent = <&intc>;
-			interrupt = <65>;
-			dmas = <&edma 16
-				&edma 17
-				&edma 18
-				&edma 19>;
-			dma-names = "tx0", "rx0", "tx1", "rx1";
-			ti,spi-num-cs = <2>;
-			status = "disabled";
-		};
-
-		spi1: spi@481a0000 {
-			compatible = "ti,omap4-mcspi";
-			ti,hwmods = "spi1";
-			#address-cells = <1>;
-			#size-cells = <0>;
-			reg = <0x481a0000 0x400>;
-			interrupt-parent = <&intc>;
-			interrupt = <125>;
-			dmas = <&edma 42
-				&edma 43
-				&edma 44
-				&edma 45>;
-			dma-names = "tx0", "rx0", "tx1", "rx1";
-			ti,spi-num-cs = <2>;
-			status = "disabled";
-		};
-
 		wdt2: wdt@44e35000 {
 			compatible = "ti,omap3-wdt";
 			ti,hwmods = "wd_timer2";
@@ -535,6 +475,11 @@
 				&edma 45>;
 			dma-names = "tx0", "rx0", "tx1", "rx1";
 			status = "disabled";
+
+		};
+
+		nop-phy {
+			compatible = "nop-xceiv-usb";
 		};
 
 		usb@47400000 {
@@ -549,7 +494,7 @@
 			num-eps = <16>;
 			ram-bits = <12>;
 			port0-mode = <3>;
-			port1-mode = <3>;
+			port1-mode = <1>;
 			power = <250>;
 			ti,hwmods = "usb_otg_hs";
 		};
-- 
1.7.10.4

